src/components/routes/chat/ChatPanel.tsx

```diff
@@ -257,9 +264,15 @@ export default function ChatPanel(props) {
 		}
 	</Box>;
 
+	const dateToday = new Date();
+
 	return (
 		<Box className={classes.flexGrow}>
-			<Box className={classes.chatPanel}>
+			<Box className={
+				(dateToday.getMonth() === 9 && dateToday.getDate() >= 30) || (dateToday.getMonth() === 10 && dateToday.getDate() === 1)
```

Comment:
This code is duplicated 3 times I believe, could you make an importable `isHalloween` function in the utils folder and call that to check whether it's Halloween or not?